rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/NSCreationServlet.java

```diff
@@ -159,12 +155,15 @@ private void createNs(HttpServletRequest request, HttpServletResponse response)
             }
 
             if (failFast != null) {
+                LOGGER.error("Cannot apply node source action: " + failFast);
+                LOGGER.error("Request parameters: ");
+                fileItems.forEach(fileItem -> LOGGER.error(fileItem.getFieldName() + "=" + fileItem.getString()));
                 throw new RestServerException(failFast);
             }
 
             String jsonResponsePayload;
             switch (nodeSourceAction) {
```

Comment:
I guess that at this level the `nodeSourceAction` is never `Unknown`, right?

Reply:
Yes, I set it to UNKNOWN to prevent NPE just in case.